Form/Type/EntityCheckboxGridType.php

```diff
@@ -130,6 +114,20 @@ private function internalConfigureOptions(OptionsResolver $resolver)
             'x_path',
             'y_path',
         ));
+
+        if (version_compare(Kernel::VERSION, '2.6.0', '>=')) {
```

Comment:
comparing the kernel version is wrong. You must detect the version of OptionsResolver being used, not the version of HttpKernel (they might be different when you don't use the full symfony package)